containers/dockerfile_gpuhoomd3conda

```diff
@@ -0,0 +1,80 @@
+FROM nvidia/cuda:11.6-devel 
```

Comment:
Have we checked that this cuda version is available on Fry?

Reply:
Good point, it looks like it isn't. I figured it would be best to try and use the latest versions of packages when possible. How difficult would it be to get a newer cuda module on Fry?

Reply:
We can ask Jason if necessary.